midx.c

```diff
@@ -1014,6 +1014,9 @@ int verify_midx_file(const char *object_dir)
 	for (i = 0; i < m->num_packs; i++) {
 		if (prepare_midx_pack(m, i))
 			midx_report("failed to load pack in position %d", i);
+
+		if (m->packs[i])
+			install_packed_git(the_repository, m->packs[i]);
```

Comment:
Why did you decide to put this here instead of `prepare_midx_pack`?  Did the other places that `prepare_midx_pack` is called cause issues?

Reply:
The packfile code is very dense (and obscure).   I tried adding it in several places, like that one.
It fixed "midx verify", but caused weird side effects with other commands, like count-objects,
that made the packed_git list circular and caused git to hang.

The verify code seems to be unique in that it needs to touch each packfile as part of the
verification process.  Whereas others just need to know about the OID values.

This place is the safest, in that it is isolated to "midx verify".  There are no functional changes
outside of "midx verify".  So while, I think this fix is correct, if it is wrong, it is limited to this
verb.